.github/workflows/build.yml

```diff
@@ -82,8 +87,8 @@ jobs:
           echo "Release Tag: ${{ steps.release.outputs.tag }}"
           echo "Deployment Release Tag: ${{ steps.deployment-phase.outputs.tag }}"
           echo "Deployment Digest: ${{ steps.get-hash.outputs.hash }}"
-      - name: set-version-tag-cate-app
-        uses: bc-org/update-application-version-tags@main
+      - name: set-version-tag-cate
```

Comment:
Why do you remove "-app" there? didn't you suggest this in the last PR?

Reply:
Sorry for not seeing this. It is the name pf the action on bc-org

Reply:
@AliceBalfanz Yes, I did suggest it, because I did not understand what it refers to. I thought it was supposed to be the app-name, but in fact it was the name of the directory in k8s-configs that should be updated. In the endit is just a name though and you can make arguments for both namings, so this is not critical.

The matter becomes clearer if you compare it to the workflow script in xcube-hub ( https://github.com/bcdev/xcube-hub/blob/master/.github/workflows/xcube-hub_workflow.yaml ) where there are two directories that need to be adjusted - and the respective sections are named set-version-tag-cate and set-version-tag-xcube-gen.